src/drt/src/io/io.cpp

```diff
@@ -831,7 +831,7 @@ void io::Parser::updateNetRouting(frNet* netIn, odb::dbNet* net)
             styleWidth = layer->getWidth();
           }
         }
-        width = (width) ? width : tech_->name2layer_[layerName]->getWidth();
+        width = tech_->name2layer_[layerName]->getWidth();
```

Comment:
Please explain this change.

Reply:
The `width` var is assigned with zero here: https://github.com/eder-matheus/OpenROAD/blob/drt_coverity/src/drt/src/io/io.cpp#L711. This is the beginning of the loop, and its value is not changed anywhere else before reaching line 834. The check is redundant because `width` will always be zero at this point.